examples/rules_ios/test/fixtures/bwb.xcodeproj/rules_xcodeproj/bazel/Debug-swift_debug_settings.py

```diff
@@ -17,7 +17,7 @@
 _TRIPLE_MATCH = re.compile(r"([^-]+-[^-]+)(-\D+)[^-]*(-.*)?")
 
 _SETTINGS = {
-	"arm64-apple-ios Lib": {
+	"arm64-apple-ios Lib.framework": {
```

Comment:
All of these static frameworks shouldn't be appearing in here. That's a future PR though.